setup.cfg

```diff
@@ -6,3 +6,6 @@ filterwarnings =
     error
     ignore:file format.*:UserWarning
     ignore:.*non-empty pattern match.*:FutureWarning
+
+[flake8]
+max-line-length = 120
```

Comment:
```suggestion
max-line-length = 100
```

Reply:
Since I am stuck at this, I'll open the flake8 change as a separate PR.